static/app-strings.json

```diff
@@ -1001,7 +1001,6 @@
   "Create Account": "Create Account",
```

Comment:
**Scenario:** new account, rewards not approved, email verified.

- [ ] I'm logged in, but subtitle still asking me to create an account (maybe legacy issue, at least the Button label 'Finish Account' makes sense).
- [ ] Pressing 'Finish Account' brings me to http://localhost:9090/$/invite/lbry://@dev-test and does nothing.  I think I should be trying to get my rewards approved first?

<img width="320" alt="image" src="https://user-images.githubusercontent.com/64950861/194987937-6e5911d4-716c-47a0-ab66-fb9559d1ca66.png">

Reply:
There were some existing bugs with this process too...not something introduced.